src/lib.rs

```diff
@@ -16,7 +16,7 @@ pub unsafe fn vsprintf<V>(format: *const c_char, va_list: *mut V) -> Result<Stri
 }
 
 /// Prints a format string into a list of raw bytes that form
-/// a null-terminated C string.
+/// a non-null-terminated list of bytes.
```

Comment:
Hmm, I'm pretty sure I didn't change the behavior here, and the original doc is wrong too :shrug: https://github.com/dylanmckay/vsprintf/pull/2/files